hpc/LoadBalancer.cpp

```diff
@@ -41,6 +41,8 @@ const std::vector<std::string> get_model_names() {
     return umbridge::SupportedModels(hq_job.server_url);
 }
 
+std::atomic<int> HyperQueueJob::job_count = 0;
```

Comment:
Could that go in the header?
Also, bit of a detail, but might be safer to go with std::atomic<std::int32_t>, since hyperqueue assumes 32bit signed integers for priority and int doesn't guarantee to be 32bit

Reply:
No, it can't. You have to define the static member in the cpp file, c.f. https://stackoverflow.com/questions/9282354/static-variable-link-error. 
Good point about the 32 bits.